extensions/sql-migration/src/dialog/validationResults/validateIrDialog.ts

```diff
@@ -0,0 +1,676 @@
+		}
+	}
+
+	private async _initializeDialog(dialog: azdata.window.Dialog): Promise<void> {
+		return new Promise<void>((resolve, reject) => {
+			dialog.registerContent(async (view) => {
+				try {
+					dialog.okButton.label = constants.VALIDATE_IR_DONE_BUTTON;
+					dialog.okButton.position = 'left';
+					dialog.okButton.enabled = false;
+					dialog.cancelButton.position = 'left';
+
+					this._disposables.push(
+						dialog.cancelButton.onClick(
+							e => {
+								this._canceled = true;
+								this._saveResults();
+								this._onClosed();
+							}));
+
+					this._disposables.push(
+						dialog.okButton.onClick(
+							e => this._onClosed()));
+
+					const headingText = view.modelBuilder.text()
+						.withProps({
+							value: constants.VALIDATE_IR_HEADING,
+							CSSStyles: {
+								'font-size': '13px',
+								'font-weight': '400',
+								'margin-bottom': '10px',
+							},
+						})
+						.component();
+					this._startLoader = view.modelBuilder.loadingComponent()
+						.withProps({
+							loading: false,
+							CSSStyles: { 'margin': '5px 0 0 10px' }
+						})
+						.component();
+					const headingContainer = view.modelBuilder.flexContainer()
+						.withLayout({
+							flexFlow: 'row',
+							justifyContent: 'flex-start',
+						})
+						.withItems([headingText, this._startLoader], { flex: '0 0 auto' })
+						.component();
+
+					this._resultsTable = await this._createResultsTable(view);
+
+					this._startButton = view.modelBuilder.button()
+						.withProps({
+							iconPath: IconPathHelper.restartDataCollection,
+							iconHeight: 18,
+							iconWidth: 18,
+							width: 100,
+							label: constants.VALIDATE_IR_START_VALIDATION,
+						}).component();
+
+					this._cancelButton = view.modelBuilder.button()
+						.withProps({
+							iconPath: IconPathHelper.stop,
+							iconHeight: 18,
+							iconWidth: 18,
+							width: 100,
+							label: constants.VALIDATE_IR_STOP_VALIDATION,
+							enabled: false,
+						}).component();
+					this._copyButton = view.modelBuilder.button()
+						.withProps({
+							iconPath: IconPathHelper.copy,
+							iconHeight: 18,
+							iconWidth: 18,
+							width: 150,
+							label: constants.VALIDATE_IR_COPY_RESULTS,
+							enabled: false,
+						}).component();
+
+					this._disposables.push(
+						this._startButton.onDidClick(
+							async (e) => await this._runValidation()));
+					this._disposables.push(
+						this._cancelButton.onDidClick(
+							e => {
+								this._cancelButton.enabled = false;
+								this._canceled = true;
+							}));
+
+					this._disposables.push(
+						this._copyButton.onDidClick(
+							async (e) => this._copyValidationResults()));
+
+					const toolbar = view.modelBuilder.toolbarContainer()
+						.withToolbarItems([
+							{ component: this._startButton },
+							{ component: this._cancelButton },
+							{ component: this._copyButton }])
+						.component();
+
+					const resultsHeading = view.modelBuilder.text()
```

Comment:
As soon as I load the validation dialog this is the first thing I see: 
![image](https://user-images.githubusercontent.com/11844501/207174786-735fb6de-fea6-4fe6-a889-db4f2c170b83.png)


The "validation step details" heading along with the text box appearing but also being empty is a little confusing. It's only when I click on one step specifically (e.g. "validating IR connectivity") does the text box populate with something useful:
![image](https://user-images.githubusercontent.com/11844501/207175158-bbe66aa7-cee5-4344-ab57-97a05924c8f9.png)


Can this be changed so that maybe this "validation step details" label and textbox are hidden entirely until an actual step is selected?

(along with this, I suppose this means we can also grey out the "Copy validation results" button until a step is selected, since similarly there's nothing to copy until a specific step is selected)

Reply:
Adding to this, I think having the mouse pointer change to the "clickable" pointer would be nice when you hover over the rows in the validation result to indicate that clicking the row would populate the validation step details box, it took me a while to figure that out